src/js/controller/controller.js

```diff
@@ -191,7 +191,7 @@ Object.assign(Controller.prototype, {
                     if (allowPreload && position && duration > 0 && position >= duration - BACKGROUND_LOAD_OFFSET) {
                         mediaModel.off('change:position', onPosition, this);
                         _programController.backgroundLoad(item, index);
-                    } else if (recsAuto) {
+                    } else if (recsAuto && !item) {
```

Comment:
my one concern with this change is i'm not 100% on the behavior of recs that are on the item not the playlist (in the `recommendations` key) - we should test this when reviewing these changes

Reply:
also this makes me think this has always been broken?

Reply:
I think this is broken after the async changes since we are passing in `index` to `_programController.backgroundLoad`, and that passed in `index` is used to get the `asyncItem`.
Before the async changes, it did not take in `index`, and we use the following to get the next item (line 184 in this PR):
```
let index = model.get('item') + 1;
let item = model.get('playlist')[index];
```